src/client/rsg_actor.cpp

```diff
@@ -15,7 +15,7 @@ using namespace ::simgrid;
 XBT_LOG_NEW_CATEGORY(RSG,"Remote SimGrid");
 XBT_LOG_NEW_DEFAULT_SUBCATEGORY(RSG_ACTOR, RSG, "RSG::Actor");
 
-rsg::Actor::Actor() : pHost(NULL) {
+rsg::Actor::Actor(unsigned long int addr) : p_remoteAddr(addr), pHost(NULL) {
```

Comment:
addr is a useless variable name. Please document its purpose in its name.
`p_remoteAddr(addr)` is really not instructive.